### DISCUSSION TITLE
Option to 'start|in' subcommand to automatically 'stop|out' running entry

### DISCUSSION THREAD

**Discussion by matleh** [jotaen/klog#267] [Feature ideas & discussions]
When I do some task and switch to another task, I currently have to `klog out` first and then `klog in -s "do stuff"`. 
It would be nice to be able to just do `klog in -c/--close -s "do stuff"` or `klog switch -s "do stuff"`. 

Maybe there already exists a way to do this and I just could not find it?

**Comment by jotaen** [comment]
Thanks for your suggestion, I agree that would be useful behaviour. Since such a feature would mostly be for convenience, I’m thinking that it’s probably easiest and most intuitive to extend `klog start` with a flag along the lines of `--close`/`-c`. You wouldn’t be able to add a summary to the previous (then closed) entry, but I think that’s okay (think: `klog stop --summary`).

**Reply by matleh** [reply]
Thanks for your positive response. For my current usage, the suggested flag would be sufficient. 

Not sure what should happen, if -c is given but no open task exists - either fail like `klog out` would or ignore silently, which has the benefit one can use the same command, no matter if there is an open task or not.
